lib/stateTransition/fetchLockTransactionOutputFactory.js

```diff
@@ -43,7 +46,17 @@ function getLockedTransactionOutputFactory(
       throw new IdentityLockTransactionNotFoundError(transactionHash);
     }
 
-    const transaction = new Transaction(rawTransaction);
+    if (enableLockTxOneBlockConfirmationFallback) {
+      // check confirmations
+      if (!rawTransaction.confirmations) {
```

Comment:
Do you mind to compare `rawTransaction.confirmations > 0` in order to make it more readable?

Reply:
I'm not sure we always have this property...